### PR TITLE
Update universite-du-quebec-a-montreal-etudes-litteraires-et-semiolog…

### PR DESCRIPTION
**Pull request by jjrondeau**
Adding some non breaking spaces between labels and variable/locator

### CODE REVIEW

File: universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl [hunk 1]
```diff
@@ -397,7 +397,7 @@
     </choose>
   </macro>
   <macro name="locator">
-    <group delimiter=" " prefix=", " suffix=".">
+    <group delimiter="&amp;#x00A0;" prefix=", " suffix=".">
```
Comment (adam3smith) [inline universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl:400]: That's not right, you're escaping the ampersand. Also, use the decimal escaping for readability
```suggestion
    <group delimiter="&#160;" prefix=", " suffix=".">
```

File: universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl [hunk 2]
```diff
@@ -450,7 +450,10 @@
   <macro name="page-chapter">
     <choose>
       <if match="none" locator="page">
-        <text variable="page" prefix="p. "/>
+        <group delimiter="&amp;#x00A0;">
```
Comment (adam3smith) [inline universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl:453]:
```suggestion
        <group delimiter="&#160;">
```

same here

**Review by adam3smith — CHANGES_REQUESTED** [review]
That didn't quite work. See suggested fixes

File: universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl (hunk 2 above)
Comment (jjrondeau) [reply universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl]: Done. Thanks.

File: universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl (hunk 1 above)
Comment (jjrondeau) [reply universite-du-quebec-a-montreal-etudes-litteraires-et-semiologie.csl]: Done.

**Review by jjrondeau — COMMENTED** [review]
Hi,
Changes were done at line 400 and 453 as suggested. Thanks.

**Comment by adam3smith** [comment]
Thanks?

**Pull request merged**